# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/grub2/welcome-cr.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Some PC BIOS' (like ASRock B550M-ITX) might have printed some stray debug number,
and don't have the inital text position at the beginning of the line:
carriage return, and shorten the message while at it.

--- grub-2.06/grub-core/kern/main.c.orig	2023-11-24 11:25:02.293274479 +0100
+++ grub-2.06/grub-core/kern/main.c	2023-11-24 11:25:32.125275949 +0100
@@ -272,7 +272,7 @@
 
   /* Hello.  */
   grub_setcolorstate (GRUB_TERM_COLOR_HIGHLIGHT);
-  grub_printf ("Welcome to GRUB!\n\n");
+  grub_printf ("\rGRUB\n");
   grub_setcolorstate (GRUB_TERM_COLOR_STANDARD);
 
   /* Init verifiers API. */
